.github/workflows/dockerHub.yml

```diff
@@ -0,0 +1,46 @@
+name: CI docker hub
+
+on: 
+  push:
+    branches:
+      - develop
+      - feature/*
+
+jobs:
+  linux:
+  
+    strategy:
+      matrix:
+        container:
+        - 'jmkerloch/rte-antares:centos8-ortools'
+        - 'jmkerloch/rte-antares:ubuntu-ortools'
+  
+    runs-on: ubuntu-latest
+    if: "!contains(github.event.head_commit.message, '[skip ci]')"
+    container: ${{ matrix.container }}
+    steps:
+    - uses: actions/checkout@v2
+    
+    - name: Configure
+      run: |
+           cmake -B _build -S src -DCMAKE_BUILD_TYPE=release -DBUILD_TESTING=ON
+
+    - name: Build
+      run: |           
+           cmake --build _build --config release -j2
```

Comment:
-j8 for consistency

Reply:
-j2 because we have only 2 processor on github action